### ISSUE TITLE
Laravel 6 Support

### ISSUE THREAD

**Issue by Kovah** [code16/sharp#173]
Will there be support for Laravel 6 in the near future? Would really like to try Sharp but the corresponding app is already migrated with Laravel 6.

Thank you very much! 😊

**Comment by dvlpp** [comment]
Yes, of course: we plan to release it very soon (even today, or most likely tomorrow)
